src/test/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/toolwindow/QuickAccessParametersStateTest.kt

```diff
@@ -0,0 +1,141 @@
+package nl.tudelft.ewi.se.ciselab.testgenie.toolwindow
+
+import com.intellij.testFramework.LightProjectDescriptor
+import com.intellij.testFramework.fixtures.CodeInsightTestFixture
+import com.intellij.testFramework.fixtures.IdeaTestFixtureFactory
+import com.intellij.testFramework.fixtures.impl.LightTempDirTestFixtureImpl
+import nl.tudelft.ewi.se.ciselab.testgenie.TestGenieDefaultsBundle
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.Arguments
+import org.junit.jupiter.params.provider.MethodSource
+import java.util.stream.Stream
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class QuickAccessParametersStateTest {
+    private lateinit var state: QuickAccessParametersState
+    private lateinit var fixture: CodeInsightTestFixture
+
+    @BeforeEach
+    fun setUp() {
+        val factory = IdeaTestFixtureFactory.getFixtureFactory()
+        val testFixture = factory.createLightFixtureBuilder(LightProjectDescriptor.EMPTY_PROJECT_DESCRIPTOR).fixture
+        fixture = factory.createCodeInsightFixture(testFixture, LightTempDirTestFixtureImpl(true))
+        fixture.setUp()
+
+        state = QuickAccessParametersState()
+    }
+
+    @AfterEach
+    fun tearDown() {
+        fixture.tearDown()
+    }
+
+    @ParameterizedTest
+    @MethodSource("valueGenerator")
+    fun testSerialise(updateValue: () -> Unit, expected: MutableList<String>) {
+        updateValue()
+        Assertions.assertThat(expected).isEqualTo(state.serializeChangesFromDefault())
```

Comment:
Why don't you just import all the assertions rather than calling `Assertion.*` every time?

Reply:
I personally like the pattern: "Assertions.assertThat(...)`.  It is more explicit and i really like to use it in this way

Reply:
Has been fixed though